### PR TITLE
feat: Added support for django3.2

### PR DESCRIPTION
**Pull request by Jawayria**
JIRA: https://openedx.atlassian.net/browse/BOM-2626

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,31 +1,14 @@
...
 ignore = D001
 
 [pycodestyle]
 exclude = .git,.tox,migrations
 max-line-length = 120
 
 [pydocstyle]
-; D101 = Missing docstring in public class
-; D200 = One-line docstring should fit on one line with quotes
-; D203 = 1 blank line required before class docstring
```
Comment (mraarif) [inline tox.ini:15]: are these removed by running the `modernizer` script?

File: tox.ini [hunk 2]
```diff
@@ -35,51 +18,52 @@ addopts = --cov scripts --cov-report term-missing --cov-report xml
...
+    py.test {posargs}
 
 [testenv:docs]
-setenv =
-	PYTHONPATH = {toxinidir}
-whitelist_externals =
-	make
-	rm
-deps =
-	-r{toxinidir}/requirements/doc.txt
```
Comment (mraarif) [inline tox.ini:53]: we should undo these changes

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -14,7 +14,7 @@ jobs:
       matrix:
         os: [ubuntu-20.04]
         python-version: ['3.8']
-        toxenv: [django22, quality, docs, pii_check]
+        toxenv: [django22, django31, django31, django32, quality, docs, pii_check]
```
Comment (mraarif) [inline .github/workflows/ci.yml:17]: did we mean to add `django30` instead of `django31` twice

File: tox.ini (hunk 1 above)
Comment (Jawayria) [reply tox.ini:15]: Yes

**Review by mraarif — APPROVED** [review]

File: scripts/renderers.py [hunk 4]
```diff
@@ -7,7 +7,6 @@
 import re
 import csv
 import logging
-from collections import OrderedDict, defaultdict, Hashable
```
Comment (awais786) [inline scripts/renderers.py:10]: Revert this change.

Comment (awais786) [reply scripts/renderers.py:10]: https://github.com/edx/edx-toggles/pull/35/files Check this PR.
